requirements-extra.txt

```diff
@@ -6,4 +6,5 @@ mlflow>=1.0.0
 test_tube>=0.7.5
 wandb>=0.8.21
 trains>=0.14.1
-matplotlib>=3.1.1
\ No newline at end of file
+matplotlib>=3.1.1
+hydra_core >=0.11.3
```

Comment:
Please pin to 0.11.3 for now.
The next version (1.0.0) will break some compatibility and it would be good to test it before you allow it.

One of the most important changes is that the supported Python versions are changing from `>= 2.7` to `>=3.6`.

I will have a release candidate for Hydra 1.0 in a few weeks, at that point, after a test - you can relax the restriction.